project/accounts/templates/accounts/register/header.html

```diff
@@ -1,8 +1,9 @@
 {% load static %}
+{% load i18n %}
 
 <div class="valign center">
     <div class="logo center">
         <object class="logo-img" type="image/svg+xml" data="{% static "img/logo-civiwiki-tree-white.svg" %}" height="64px"></object>
-        <span class="wordmark">CiviWiki</span>
+        <span class="wordmark">{% trans "CiviWiki" %} </span>
```

Comment:
CiviWiki is a proper noun, so doesn't need to be translated.
```suggestion
        <span class="wordmark">CiviWiki</span>
```